app/views/admin_mailer/new_critical_software_updates.text.erb

```diff
@@ -1,3 +1,7 @@
 <%= raw t('admin_mailer.new_critical_software_updates.body') %>
 
+<% @software_updates.each do |update| %>
+- Mastodon #{update.version}: #{update.release_notes}
```

Comment:
This line could be `- #{update.to_s}` (or `to_description_s` or whatever...) both for reuse between these two mailers, and to keep the knowledge of building the desc string either in the model or a presenter?